src/ext_localconf.php

```diff
@@ -82,39 +82,91 @@
     )
 );
 
+\TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin(
+    'EWW.' . $_EXTKEY,
+    'rest_api',
+    [
+        'Api' => 'list, show, create, suggestion, importDoiWithoutSaving, importPubmedWithoutSaving, importIsbnWithoutSaving, importBibtexWithoutSaving, importRisWithoutSaving, addFisId',
+    ],
+    [
+        'Api' => 'list, show, create, suggestion, importDoiWithoutSaving, importPubmedWithoutSaving, importIsbnWithoutSaving, importBibtexWithoutSaving, importRisWithoutSaving, addFisId',
+    ]
+);
+
+// Add RealURL for api configuration
+if (\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::isLoaded('realurl')) {
+//    $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['realurl']['_DEFAULT']['fixedPostVars']['api'] = [
```

Comment:
Is this important for the deployment? Why is it commented out?

Reply:
This was originally required for the realurl configuration of the fis api. But since we can do without realurl paths in version 4, as discussed in one of our meetings, I have now removed the code accordingly.
In the future we will use the TYPO3 9 core solution for speaking paths.